Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -21,14 +21,11 @@
         </Target>
     </ECRelationshipClass>
 
-    <ECEntityClass typeName="DimensionsAspect" description="An aspect that defines basic object measurements: height, length width." >
+    <ECEntityClass typeName="DimensionsAspect" displayLabel="Dimensions Aspect" description="An aspect that defines basic object measurements: height, length width." >
         <BaseClass>QuantityTakeoffAspect</BaseClass>
-        <ECCustomAttributes>
-            <ClassHasHandler xmlns="BisCore.01.00.00"/>
-        </ECCustomAttributes>
-        <ECProperty propertyName="Height" typeName="double" kindOfQuantity="AECU:LENGTH" description="Object's height/depth in meters."/>
-        <ECProperty propertyName="Length" typeName="double" kindOfQuantity="AECU:LENGTH" description="Object's length in meters."/>
-        <ECProperty propertyName="Width" typeName="double" kindOfQuantity="AECU:LENGTH" description="Object's width/thickness in meters."/>
+        <ECProperty propertyName="Height" typeName="double" displayLabel="Height" kindOfQuantity="AECU:LENGTH" description="Object's height/depth in meters."/>
+        <ECProperty propertyName="Length" typeName="double" displayLabel="Length" kindOfQuantity="AECU:LENGTH" description="Object's length in meters."/>
+        <ECProperty propertyName="Width" typeName="double" displayLabel="Width" kindOfQuantity="AECU:LENGTH" description="Object's width/thickness in meters."/>
```

Comment:
There is no point in adding displayLabel attributes if they are the same as the invariant name ... it doesn't hurt much but does incrementally increase the xml size and memory footprint of the schema.  Both increases are small but every little bit counts.